### ISSUE TITLE
Generate linux binaries for armhf and arm64

### ISSUE THREAD

**Issue by apandada1** [sharkdp/insect#363]
How do you compile the binaries in releases page?  Can you generate binaries for arm in the next release?

Also, can you post instructions about how to compile these binaries?

**Comment by mhmdanas** [comment]
See https://github.com/sharkdp/insect/issues/297 and #298.

Tl;DR we don't build binaries for Insect anymore, as the tool we used to generate them (Nexe) bundled very old Node versions. If you have any alternatives, feel free to suggest them.

However, note that Insect's code ultimately compiles down to JavaScript, so as long as you have Node 12 or later, you can just build Insect and run the generated JS file. To do so, clone this repo, then run `npm install && npm run build`. This will produce an `index.cjs` file, which you should be able to run by just running `./index.cjs`. Strictly speaking, it is not a binary, but a file containing minified and bundled JS, but it should work nonetheless.

If you face any issues with this, feel free to post them here.
